src/genome_builds.php

```diff
@@ -167,7 +167,14 @@
                 }
 
                 if ($bToAdd) {
-                    $_DB->query(rtrim($sSQL, ','));
+                    $_DB->query(
+                        ($sTable != TABLE_VARIANTS?
+                            rtrim($sSQL, ',') :
+                            $sSQL .
+                            ' ADD INDEX (chromosome, position_g_start_' . $_POST['column_suffix'] .
+                            ', position_g_end_' . $_POST['column_suffix'] . ')'
+                        )
```

Comment:
I think it's a bit more readable to make this into a "normal" `if()`, run before the `$_DB->query()` command.